frontend/pages/topics/[slug].tsx

```diff
@@ -22,12 +68,19 @@ const TopicDetailPage: FC<TopicProps> = ({ topic }: TopicProps) => {
         {topic.description && <p dangerouslySetInnerHTML={{ __html: topic.description }} />}
         {topic.propositions &&
           topic.propositions.map((proposition) => (
-            <Link href={`/propositions/${proposition.slug}`} key={proposition.slug}>
-              <a>
```

Comment:
We probably don't want to remove this anchor. [`Link`](https://nextjs.org/docs/api-reference/next/link)'s documentation mentions that SEO might be harmed by not rendering an anchor, I think because the `href` doesn't appear otherwise.